lib/cog/command/pipeline/executor.ex

```diff
@@ -785,4 +806,15 @@ defmodule Cog.Command.Pipeline.Executor do
     |> Enum.sort
   end
 
+  defp maybe_warn_on_ambiguous_aliases(%Ast.Pipeline{invocations: invocations}, state) do
+    Enum.map(invocations, fn(invocation) ->
+      case AmbiguousAliasChecker.ambiguous_command?(invocation) do
+        {true, alias_name} ->
+          Helpers.send_reply("Warning: The alias '#{alias_name}' and the command '#{invocation.command}' are ambiguous. Ignoring '#{alias_name}' and executing '#{invocation.command}'. Consider renaming your alias.", state.request, state.mq_conn)
```

Comment:
We fail with a `SemanticError` if an unqualified-command name is given, which is the same kind of ambiguity. We should always fail in such a case.